docs/streams/developer-guide.html

```diff
@@ -1014,13 +1014,819 @@
 
 
     <h5><a id="streams_dsl_transformations_stateful" href="#streams_dsl_transformations_stateful">Stateful transformations</a></h5>
-    Stateless transformations, by definition, do not depend on any state for processing, and hence implementation-wise they do not
-    require a state store associated with the stream processor.Stateful transformations, by definition, depend on state for processing
-    inputs and producing outputs, and hence implementation-wise they require a state store associated with the stream processor. For
-    example, in aggregating operations, a windowing state store is used to store the latest aggregation results per window; in join
-    operations, a windowing state store is used to store all the records received so far within the defined window boundary.
+    <h6><a id="treams_dsl_transformations_stateful_overview" href="#streams_dsl_transformations_stateful_overview">Overview</a></h6>
+    <p>
+        Stateful transformations, by definition, depend on state for processing inputs and producing outputs, and
+        hence implementation-wise they require a state store associated with the stream processor. For example,
+        in aggregating operations, a windowing state store is used to store the latest aggregation results per window;
+        in join operations, a windowing state store is used to store all the records received so far within the
+        defined window boundary.
+    </p>
+    <p>
+        Note, that state stores are fault-tolerant. In case of failure, Kafka Streams guarantees to fully restore
+        all state stores prior to resuming the processing.
+    </p>
+    <p>
+        Available stateful transformations in the DSL include:
+    <ul>
+        <li><a href=#streams_dsl_aggregations>Aggregating</a>
```

Comment:
closing `</li>` tags missing here and elsewhere

Reply:
> An li element's end tag may be omitted if the li element is immediately followed by another li element or if there is no more content in the parent element. https://www.w3.org/TR/2014/REC-html5-20141028/syntax.html#optional-tags

Reply:
I'll add them anyway.